interfaces/python/CMakeLists.txt

```diff
@@ -15,11 +15,6 @@ get_directory_property(hasParent PARENT_DIRECTORY)
 if(NOT hasParent)
     message(STATUS "Standalone Python interface build")
 
-    # find_package(<PackageName>) searches prefixes given in <PackageName>_ROOT CMake variable
-    if(POLICY CMP0074)
-        cmake_policy(SET CMP0074 NEW)
```

Comment:
This should be replaced by:
```
cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
```

For standalone builds, the policy set in the parent will not apply.